src/test/HostActivationTests/HostActivationTests.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <AssemblyName>HostActivationTests</AssemblyName>
+    <PackageId>HostActivationTests</PackageId>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
```

Comment:
This is just what dotnet migrate spits out. Ditto for a lot of other comments. I'll try to clean up these things where I see them.